tst/providers/MeetingProvider/index.test.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  LogLevel,
+  EventName, 
+  EventAttributes, 
+  MeetingSessionCredentials, 
+  MeetingSessionURLs
+} from 'amazon-chime-sdk-js';
+
+import MeetingManager from '../../../src/providers/MeetingProvider/MeetingManager';
+import { MeetingJoinData } from '../../../src/providers/MeetingProvider/types';
+
+
+describe('Meeting Provider', () => {
+
+  it('events are received correctly', async (done) => {
```

Comment:
Why do we have both `async` and done?

Reply:
Async for small waits between events, and done to ensure events are actually sent